modules/jpa/src/main/java/io/oasp/module/jpa/common/api/to/SearchCriteriaTo.java

```diff
@@ -21,7 +21,7 @@
   /** @see #getPagination() */
   private PaginationTo pagination;
 
-  /** @see getSearchTimeout */
+  /** @see #getSearchTimeout */

```

Comment:
I did not find any of the fixed JavaDoc errors with help of my IDE. Actually, it was the `mvn site` plugin which complained about these errors.